tmva/pymva/test/TestRModelParserKeras.C

```diff
@@ -411,6 +412,45 @@ TEST(RModelParser_Keras, ACTIVATIONS)
     }
 }
 
+TEST(RModelParser_Keras, SWISH)
+{
+    constexpr float TOLERANCE = DEFAULT_TOLERANCE;
+    float input[]={1,1,1,1,1,1,1,1};
+    TMVA_SOFIE_swish_model::Session s("KerasSwish_model.dat");
+    std::vector<float> output = s.infer(input);
+
```

Comment:
Many of the Keras tests, especially on the activation functions work in the same way, we can maybe try to generalize them to reduce code repetitions.

Reply:
Yes, we should do this, but it can be a separate PR